### ISSUE TITLE
Unlike train images, test images not resized by the network

### ISSUE THREAD

**Issue by atique81** [Russell91/ReInspect#40]
Dear @Russell91 

while the train.py script does resize the train images to 640x480, test images are not resized (https://github.com/Russell91/ReInspect/blob/master/train.py#L37), and therefore, the evaluation ipython script complains for images of size different than 640x480. So, do I just resize the test images using scipy.misc.imresize() to 640x480 (after train.py#L37), or I need to resize it in a different way?

**Comment by Russell91** [comment]
Yeah, you can resize using any function. You can also just edit the image
size to some multiple of 32 in the json file, and the resize will happen
dynamically. This will be a little slower, but more convenient.

Best,
Russell

**Comment by atique81** [comment]
@Russell91, thank you so much for the reply. I am just wondering, how is it possible to subtract the image mean of size 640X480 from a test image of different size, even though the image may be an integer multiple of 32 (https://github.com/Russell91/ReInspect/blob/master/train.py#L37).
